### PR TITLE
[Security] Admin Authenticated Arbitrary JavaScript-CSS File Read

### PR DESCRIPTION
**Pull request by robertSt7**

### CODE REVIEW

File: bundles/AdminBundle/Controller/Admin/MiscController.php [hunk 1]
```diff
@@ -137,8 +137,18 @@ public function scriptProxyAction(Request $request)
...
-                $scriptPath = PIMCORE_WEB_ROOT . $request->get('scriptPath');
+            $scriptPath = $request->get('scriptPath');
+            if ($scriptPath) {
+                $scriptPath = realpath(PIMCORE_WEB_ROOT . $scriptPath);
+                if(!$scriptPath) {
+                    throw $this->createNotFoundException('Directory not found!');
+                }
+
+                if(!str_starts_with($scriptPath, rtrim(str_replace('../', '', PIMCORE_PROJECT_ROOT), './'))) {
+                    throw $this->createAccessDeniedException('Scripts not found! Please do not navigate out of the web root directory!');
```
Comment (blankse) [inline bundles/AdminBundle/Controller/Admin/MiscController.php:148]: PIMCORE_PROJECT_ROOT is not the "web root directory".
Should be changed to "project root directory"

Comment (jdreesen) [reply bundles/AdminBundle/Controller/Admin/MiscController.php:148]: Is this even right? Do we really want to allow any file from the project to be included in the script?

Comment (brusch) [reply bundles/AdminBundle/Controller/Admin/MiscController.php:148]: No, this is not right, this needs to be changed to `PIMCORE_WEB_ROOT` ... 
@robertSt7 can you please create a follow up?

Comment (brusch) [reply bundles/AdminBundle/Controller/Admin/MiscController.php:148]: We could even further restrict it to files ending with `.js` `.css` ...

Comment (robertSt7) [reply bundles/AdminBundle/Controller/Admin/MiscController.php:148]: is solved here: https://github.com/pimcore/pimcore/pull/14970

**Pull request merged**